CHANGELOG.md

```diff
@@ -3,6 +3,53 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 6.2.0
+
+_Apr 13, 2023_
+
+We'd like to offer a big thanks to the 8 contributors who made this release possible. Here are some highlights ✨:
+
+- Add `@mui/base` as a `peerDependency` of `@mui/x-date-pickers` and `@mui/x-date-pickers-pro`
+
+  Both libraries were not working correctly if used without `@mui/base`.
+  Most package manager should automatically use the `@mui/base` version installed for `@mui/material`.
```

Comment:
Maybe it would also make sense having the a short section on inclusion of `formatDensnity` prop as well as the "changed default"? 🤔

Reply:
Something like that ?

```suggestion
  Most package manager should automatically use the `@mui/base` version installed for `@mui/material`.
  
  - The value rendered in the input no longer has spaces around the `/` characters. 
  
    You can use the `formatDensity='spacious'` prop to add it back.
    More information on [the dedicated doc section](https://mui.com/x/react-date-pickers/custom-field/#change-the-format-density)
```

Reply:
Yup, sounds good! 👍